contracts/Issuehunter.sol

```diff
@@ -388,8 +388,8 @@ contract Issuehunter is Mortal {
     // calculation at a higher precision.
     //
     // TODO: review math
-    function _reciprocalPerMille(uint amount, uint tipPerMille) internal returns (uint) {
-        return amount * (1000 - tipPerMille) / 1000;
+    function _reciprocalPerMille(uint256 amount, uint tipPerMille) internal returns (uint) {
+        return amount * (uint256(1000 - tipPerMille)) / 1000;
```

Comment:
This amount is used in computations that will be transferred, doesn't it need to be a uint256 as well?

Reply:
Yes make sense! It'd be a small amount this number but anyway better to declare it as a uint256